recruiter/tests.py

```diff
@@ -46,20 +83,58 @@ def test_get_all_recruiters_of_a_specified_company_returns_all_recruiters_as_a_q
 
 @pytest.mark.django_db
 def test_if_user_is_a_recruiter_returns_true_if_the_auth_user_is_associated_with_a_recruiter_account(example_recruiter):
-    example_user_not_recruiter = User.objects.create_user("notrecruiter", "password")
+    example_user_not_recruiter = User.objects.create_user(
+        "notrecruiter", "password")
     example_user_not_recruiter.save()
     assert Recruiter.is_recruiter(example_user_not_recruiter.id) is False
     assert Recruiter.is_recruiter(example_recruiter.user.id) is True
 
 
 @pytest.mark.django_db
 def test_get_user_as_recruiter_object(example_recruiter):
-    assert isinstance(Recruiter.get_recruiter(example_recruiter.user.id), Recruiter)
+    assert isinstance(Recruiter.get_recruiter(
+        example_recruiter.user.id), Recruiter)
 
 
 @pytest.mark.django_db
 def test_get_recruiter_raises_exception_for_non_recruiter_user():
-    example_user_not_recruiter = User.objects.create_user("notrecruiter", "password")
+    example_user_not_recruiter = User.objects.create_user(
+        "notrecruiter", "password")
     example_user_not_recruiter.save()
     with pytest.raises(Recruiter.DoesNotExist):
         Recruiter.get_recruiter(example_user_not_recruiter.id)
+
+
+@pytest.mark.django_db
+def test_getting_the_correct_url(rf, admin_user):
+    request = rf.get('/recruiterRegister/')
+    # Remember that when using RequestFactory, the request does not pass
+    # through middleware. If your view expects fields such as request.user
+    # to be set, you need to set them explicitly.
+    # The following line sets request.user to an admin user.
+    request.user = admin_user
+    response = recruiterRegister(request)
+    assert response.status_code == 200
```

Comment:
This test is messy:

1. The use of "tf" makes it harder to figure out what the test is doing and I don't think its necessary.
2. You only assert on the HTTP response code, and not the content. Is the system displaying the right form?
3. The test name does not indicate what is the expected behavior.